instrumentation/aws-sdk/aws-sdk-1.11/testing/src/main/groovy/io/opentelemetry/instrumentation/awssdk/v1_11/AbstractSqsTracingTest.groovy

```diff
@@ -141,17 +138,15 @@ abstract class AbstractSqsTracingTest extends InstrumentationSpecification {
             "aws.queue.url" "http://localhost:$sqsPort/000000000000/testSdkSqs"
             "rpc.system" "aws-api"
             "rpc.service" "AmazonSQS"
-            "http.method" "POST"
-            "http.status_code" 200
-            "http.url" "http://localhost:$sqsPort"
-            "net.peer.name" "localhost"
-            "net.peer.port" sqsPort
+            "$SemanticAttributes.HTTP_REQUEST_METHOD" "POST"
+            "$SemanticAttributes.HTTP_RESPONSE_STATUS_CODE" 200
+            "$SemanticAttributes.URL_FULL" "http://localhost:$sqsPort"
+            "$SemanticAttributes.SERVER_ADDRESS" "localhost"
+            "$SemanticAttributes.SERVER_PORT" sqsPort
             "$SemanticAttributes.MESSAGING_SYSTEM" "AmazonSQS"
             "$SemanticAttributes.MESSAGING_DESTINATION_NAME" "testSdkSqs"
             "$SemanticAttributes.MESSAGING_OPERATION" "receive"
-            "$SemanticAttributes.NET_PROTOCOL_NAME" "http"
```

Comment:
There are other tests that explicitly test that we're not setting the held back attributes. Probably not important, just pointing it out that this doesn't do that. I think it's fine.

Reply:
It actually does -- the groovy `attributes {}` section detects extra attributes and throws if anything not explicitly asserted appears.

Reply:
Woah, good to know! Thanks.